### PR TITLE
Remove unused document signing code.

### PR DESCRIPTION
**Pull request by gokaysatir**
Change-Id: I516e7ca947c5e9f9bded071c1c3830013012ceaa

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: coolwsd.xml.in [hunk 1]
```diff
@@ -195,10 +194,6 @@
       <enable_metrics_unauthenticated desc="When enabled, the /cool/getMetrics endpoint will not require authentication." type="bool" default="false">false</enable_metrics_unauthenticated>
     </security>
 
-    <certificates>
```
Comment (timar) [inline coolwsd.xml.in:198]: This is unrelated, and important to keep. User can set NSS database to validate signatures. Without this, signatures cannot be validated.

**Review by timar — CHANGES_REQUESTED** [review]
I had 1 inline comment. 
Also, I think @quikee wanted to keep some parts. E.g. there is the general `signdocument` command that signs the current document with the input certificate and private key. It could be kept and be a base of future development. But I need a 2nd opinion from @quikee.

File: coolwsd.xml.in (hunk 1 above)
Comment (gokaysatir) [reply coolwsd.xml.in]: Thanks for the review :) Done.

**Comment by gokaysatir** [comment]
> I had 1 inline comment. Also, I think @quikee wanted to keep some parts. E.g. there is the general `signdocument` command that signs the current document with the input certificate and private key. It could be kept and be a base of future development. But I need a 2nd opinion from @quikee.

Thank you, let me wait the review from @quikee, then i can continue according to his future plans.

**Comment by quikee** [comment]
Hi, I think it's probably fine to remove all this. The code is useful but it is written for narrow use case specifically for Verign, so we would needed to rewrite it anyway and it makes not much difference if we remove it now and come back to it when we need it or keep it.

**Review by quikee — APPROVED** [review]
Looks fine to me
